source/manual/pagerduty.html.md

```diff
@@ -23,7 +23,7 @@ and that they're correctly scheduled in.
 When an alert that triggers PagerDuty goes off, someone on the escalation schedule must acknowledge
 them, otherwise they will be escalated further. NB, 2nd line shadowers are not required to be on PagerDuty.
 
-## PagerDuty drill
+## PagerDuty drill
```

Comment:
Was this change deliberate? I'm slightly concerned about merging as the heading is already rendering correctly: https://docs.publishing.service.gov.uk/manual/pagerduty.html#pagerduty-drill

Reply:
It was [automatic](https://github.com/issyl0/dotfiles/blob/master/.vimrc#L33), which is odd because the header is currently rendering correctly. :shrug:

Reply:
Maybe our header rendering is less brittle than GitHub's?